### ISSUE TITLE
Looking for maintainers

### ISSUE THREAD

**Issue by tleunen** [tleunen/react-mdl#414]
Hi all,

For those who are users of react-mdl, or just watching for the next versions/fixes, you should have noticed a bit the lack of updates recently.

v2 of MDL is coming soon (at least the alpha/beta) and we'll need to update react-mdl to match the new components. But I'm missing time for that. Is anybody willing to jump on board and help with all this?

I'm also willing to transfer the repo to someone else, or even create an organization around react-mdl if people want. Having an organization would allow to have an easy way to show non-official components, like https://github.com/tleunen/react-mdl/issues/413 or https://github.com/tleunen/react-mdl/issues/368 for example.

I'm still interested in continuing the improvement of this lib, but I currently don't have the time and resources to do so, so hopefully a few people can help with this.

cc @HriBB @igorsantos07 @faergeek @Download

**Comment by mafzst** [comment]
Hi @tleunen!

Thanks for the work already accomplished. I use it at work for our next product and we love it.

I'm interested in to participate. I prefer joining an organization like you suggest, I have no enough time to take it on my own.

Hop I can help

Cheers

**Comment by faergeek** [comment]
@rpominov Did you take a look at v2 of MDL? Could we use it as it is without hacks here https://github.com/whitescape/react-admin-components ? If so, we could join organisation. Even if not, we could make it so 👍

**Comment by vladgolubev** [comment]
Organization is the best option, IMO. Invite some top contributors and let the project evolve.

**Comment by tleunen** [comment]
Anyone willing to take the lead?
I can create an org and invite several people but having someone to really drive the project might be best as well

**Comment by HriBB** [comment]
I have the same problem as you @tleunen. I simply don't have enough time to lead, but I will help with the V2 rewrite.

**Comment by idancali** [comment]
Hi @tleunen I am currently evaluating different React frameworks, I've used react-toolbox and material-ui and I'm at a point where I'm debating whether I should create a brand new framework from scratch or fork some existing one, etc. I'm leaning towards one that makes good use of CSS Modules and SASS, there's several reasons why I think that's important. If I were to take on the lead on your project, would you be open to moving the next version towards a CSS Modules & SASS direction?

**Comment by tleunen** [comment]
Hi @idancali css modules were indeed in the plan. For the sass direction, it really depends. For the core MDL, I didn't plan to write any css because everything should already be done by the core MDL team. But for new "plugins" (eg. components which don't exist in MDL), yep, why not...

It would be great if you're interested in taking the lead on the project make the transition to MDL v2 ;)

**Comment by idancali** [comment]
hey @tleunen the sass direction would be mainly for theming, which I've found to be a major concern in most react ui frameworks and that is really a major consideration when developers pick a framework, so I would love to have the option to theme my react-mdl components with a higher degree of freedom. But this is all stuff that we can workout as we move forward.

so yeah I'm interested in the taking the lead on the project and I'm looking forward to get started, the timing is perfect, please let me know how you want to proceed next

**Comment by tleunen** [comment]
That's great! I will create an organization and move the repo there, and then invite you and some other contributors. Any suggestions on the name? Just use `react-mdl` as well for the name?

**Comment by idancali** [comment]
yeah let's go for react-mdl I think that makes the most sense, let me know if you need help with anything during the migration

**Comment by DirtyHairy** [comment]
Chiming in: while I don't have enough spare time to lead, I will definitely help and contribute to a v2 rewrite.
